src/Datagrid/Pager.php

```diff
@@ -23,6 +24,8 @@
  */
 class Pager extends BasePager
 {
+    private const CONCAT_SEPARATOR = '|';
```

Comment:
Use separator in `CONCAT()` function for correct determinate similar records like:

| id1 | id2 |
| -- | -- |
| A | AA |
| AA | A |

| concat | concat with separator |
| -- | -- |
| AAA | A\|AA |
| AAA | AA\|A |

Reply:
Could it make sense to make the const public in case someone extends the `Pager` ?

Reply:
I think yes. I changed it.

Reply:
Thanks